test/cel-validation/backendtrafficpolicy_test.go

```diff
@@ -151,6 +151,47 @@ func TestBackendTrafficPolicyTarget(t *testing.T) {
 				"spec.targetRef: Invalid value: \"object\": this policy does not yet support the sectionName field",
 			},
 		},
+		{
```

Comment:
can you add a test which LoadBalancer type is `LeastRequest` and leave ConsistentHash nil.